./nodes is not exported

<!-- Ignoring this template may result in your bug report getting deleted -->

**Describe the bug**

When importing `WebGPURenderer` I get the following error.

```sh
Module not found: Package path ./nodes is not exported from package /node_modules/three (see exports field in /node_modules/three/package.json)
```

**To Reproduce**

Steps to reproduce the behavior:
Three version 0.144.0

Import `WebGPURenderer` like so.
`import WebGPURenderer from "three/examples/jsm/renderers/webgpu/WebGPURenderer.js";`

**Cause:**

It seems to break when there is an import from `three/nodes`. This refers to Nodes.js at three/examples/jsm/nodes/Nodes.js. However this doesn't seem to be implemented in the builds or the [exports in the package.json](https://github.com/mrdoob/three.js/blob/dev/package.json#L8).

A quick fix seems to add it to the exports in the package.json.

--- a/package.json
+++ b/package.json
@@ -13,7 +13,8 @@
     "./examples/fonts/*": "./examples/fonts/*",
     "./examples/jsm/*": "./examples/jsm/*",
     "./addons/*": "./examples/jsm/*",
-    "./src/*": "./src/*"
+    "./src/*": "./src/*",
+    "./nodes": "./examples/jsm/nodes/Nodes.js"
   },
   "repository": {
     "type": "git",

